proto/manageusers.proto

```diff
@@ -7,6 +7,13 @@ service ManageUsers {
   rpc ListAllUsers(EmptyParams) returns (stream User) {}
   rpc ListAllGroups(EmptyParams) returns (stream Group) {}
   rpc DeleteUser(WhichUser) returns (GenericError) {}
+  rpc RenameUser(RenamedUser) returns (GenericError) {}
+  rpc SetGroup(SetGroupParms) returns (GenericError) {}
```

Comment:
All looks good except maybe rename GenericError to GenericResult?  GenericError implies that it is an error no matter what the response is.

Reply:
ok, renamed